src/components/canvas.js

```diff
@@ -39,11 +39,27 @@ const canvasView = widget((update) => {
     </div>
   `
 
+  function renderCanvas () {
```

Comment:
More of a question than an review...

Why is it, at least in this case, better to refactor the function (callRender) which takes arguments to a function (renderCanvas) that does not take any arguments ?